sql/src/main/java/io/crate/analyze/QuerySpec.java

```diff
@@ -240,8 +241,35 @@ public Void visitRelationColumn(RelationColumn relationColumn, SubsetContext con
         }
     }
 
+    public QuerySpec copy() {
```

Comment:
pls add comment that not everything is copied (e.g. where clause) because its not needed